Tests/test_integration.py

```diff
@@ -29,8 +29,11 @@ def __get_integration_config(client, integration_name):
     body = {
         'page': 0, 'size': 100, 'query': 'name:' + integration_name
     }
-    res_raw = demisto_client.generic_request_func(self=client, path='/settings/integration/search',
-                                                  method='POST', body=body)
+    try:
+        res_raw = demisto_client.generic_request_func(self=client, path='/settings/integration/search',
+                                                      method='POST', body=body)
+    except ApiException as conn_error:
```

Comment:
It should only effect the specific test since its part of a looping function. I'm adding a `return None` after the exception catch. It should make sure we drop out of the process and move to the next test without bumping into the case on line 38.